ekf/sensors_client.c

```diff
@@ -9,20 +9,24 @@
 #include "tools/rotas_dummy.h"
 
 
-enum sensor_fd {client_imu, client_baro, client_termo, client_gps};
+enum sensor_fd { client_imu,
+	client_baro,
+	client_termo,
+	client_gps };
```

Comment:
The previous version was more compliant with our style.
```suggestion
enum sensor_fd { client_imu, client_baro, client_termo, client_gps };
```

Reply:
but i am not sure if clang-format will not complain :P